src/bundle/Resources/public/js/alloyeditor/src/widgets/ez-custom-tag-base.js

```diff
@@ -65,6 +66,11 @@ const customTagBaseDefinition = {
     },
 
     init: function() {
+        // Converted from eZ XML custom tags might miss some required classes
```

Comment:
From my point of view, it seems a bit odd to fix the symptom here, if the actual problem is that something is missing in the xmltext -> richtext conversion ?

Reply:
I agree with @vidarl. Seems like this not the proper way of handling this. `admin-ui` should not have any check if something was properly converted from eZ XML.